scripts/ini_template_generator.py

```diff
@@ -0,0 +1,221 @@
+"""
+Usage:
+    ini_template_generator.py <output.ini> [--force] [--no-log-file]
+    ini_template_generator.py -h | --help
+
+Description:
+    Creates a new configuration template for the Ice Road LiDAR Pipeline.
+
+Arguments:
+    <output.ini>
+        Target path for the generated INI file. This may be:
+          • An absolute path:      /abs/path/run01.ini
+          • A relative path:       configs/run01.ini  (relative to current working directory)
+          • A bare filename:       run01.ini          (saved in current working directory)
+
+Options:
+    --force
+        Overwrite the file if it already exists (skips confirmation).
+    --no-log-file
+        Disable writing to ini_template_generator.log (console output only).
+    -h, --help
+        Show this help message and exit.
+
+Examples:
+    python ini_template_generator.py my_config.ini
+    python ini_template_generator.py configs/my_config.ini --force
+    python ini_template_generator.py my_config.ini --no-log-file
+    python ini_template_generator.py --help
+"""
+
+from __future__ import annotations
+
+import sys
+import logging
+from datetime import datetime
+from pathlib import Path
+
+
+# ----------------------------------------------------------------------
+# Template Definition
+# ----------------------------------------------------------------------
+
+TEMPLATE = """# ==============================================================================================================================================
```

Comment:
Instead of having the TEMPLATE as a constant in here, I am wondering if it would be better to have it as a separate file. This separates code logic from package data.